mllib/src/main/scala/org/apache/spark/ml/util/SchemaUtils.scala

```diff
@@ -34,10 +34,14 @@ object SchemaUtils {
    * @param colName  column name
    * @param dataType  required column data type
    */
-  def checkColumnType(schema: StructType, colName: String, dataType: DataType): Unit = {
+  def checkColumnType(schema: StructType,
+                      colName: String,
```

Comment:
Please follow the Spark code style guide and use 4-space indentation for arguments: https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide